src/app/services/storage.service.ts

```diff
@@ -47,6 +47,12 @@ export interface Config {
   color?: string;
 }
 
+export interface General {
```

Comment:
We are not using this anywhere?

Reply:
i added this, to build up the structure that we are using for the entire localStorage. Right now, there is no direct way to see what is being cache from the high level view.
you created interface for "User", which is very good way to trace what is being cached and what isn't. 

Other than that, we lost track of the "general" view of the entire cache, so even experienced developer may find it hard to really sure what can be used or add in the highest level of the cache.